tests/README.md

```diff
@@ -0,0 +1,19 @@
+In order to create a new test, you need three things:
+  1. a directory wake/tests/category/testname
+  2. an empty `.wakeroot` file in that directory
```

Comment:
If we end up writing a lot of tests, the boiler plate of creating .wakeroot files could be part of the test run

Reply:
That's an option, but it also means you won't be able to run the pass.sh script by hand any more. One could alternatively include a wake --init call in the script. What's your preference? Mine was this PR in its current form.